pages/installation/docker.rst

```diff
@@ -96,6 +98,9 @@ Example::
         # Password: admin
         - GRAYLOG_ROOT_PASSWORD_SHA2=8c6976e5b5410415bde908bd4dee15dfb167a9c873fc4bb8a81f6f2ab448a918
         - GRAYLOG_WEB_ENDPOINT_URI=http://127.0.0.1:9000/api
+      links:
+        - mongodb:mongo
+        - elasticsearch
```

Comment:
[`depends_on`](https://docs.docker.com/compose/compose-file/#depends_on) and [`links`](https://docs.docker.com/compose/compose-file/#links) pretty much do the same thing.

The example files are working for me. Do you have specific problems with them?

Also see: https://docs.docker.com/engine/userguide/networking/default_network/dockerlinks/